### PR TITLE
Added Nvidia Prime Render Option to Settings

### LINKED ISSUE #369: Enable Nvidia Prime Render Offload/Prime-run
**Is your feature request related to a problem? Please describe.**
For us hybrid graphics laptops owners, we need to add `__NV_PRIME_RENDER_OFFLOAD=1  __GLX_VENDOR_LIBRARY_NAME=nvidia` to the global environment variables to force the use of the discrete GPU.

**Describe the solution you'd like**
We could have options alongside MangoHud, GameMode ... to enable Nv Prime Render Offload or to prime-run a game. Example of Lutris:
Screenshot from 2021-05-21 13-00-35

**Describe alternatives you've considered**
The current alternative is to set environment variables, but it makes quite a lot of mess there and it would be cleaner and easier to manage to have just a switch to flip.

**Additional context**

### PR DESCRIPTION
**Pull request by flavioislima**
Added new option on Global and GameSettings to enable the Nvidia Prime render as requested on #369.

Fixes #369.

### CODE REVIEW

**Review by adityaruplaha — COMMENTED** [review]
Code-wise looks okay, but I would strongly recommend not adding the stub text to all the locales when it's already passed as the default in t(). I have a feeling that it might cause issues with Weblate.

**Comment by flavioislima** [comment]
> Code-wise looks okay, but I would strongly recommend not adding the stub text to all the locales when it's already passed as the default in t(). I have a feeling that it might cause issues with Weblate.

No, weblate checks that the translations are the same as english one and it warns that the text wasn't translated. What we need to do is to merge weblate changes first. Then the changes with new labels, them weblate will find them.

File: electron/games.ts [hunk 1]
```diff
@@ -298,6 +299,7 @@ class LegendaryGame implements Game {
       audio: audioFix ? `PULSE_LATENCY_MSEC=60` : '',
       fps: showFps ? `DXVK_HUD=fps` : '',
       other: otherOptions ? otherOptions : '',
+      prime: nvidiaPrime ? '__NV_PRIME_RENDER_OFFLOAD=1 __GLX_VENDOR_LIBRARY_NAME=nvidia' : '',
```
Comment (adityaruplaha) [inline electron/games.ts:302]: I think there was another related issue where different environment variables had to be used. Not sure though.

**Review by adityaruplaha — APPROVED** [review]
Looks good then.

File: electron/games.ts (hunk 1 above)
Comment (flavioislima) [reply electron/games.ts:302]: True. But in this case we will use the same behavior as lutris since I think they know best. 👍

**Pull request merged**